src/concrete/ml/onnx/convert.py

```diff
@@ -255,7 +255,7 @@ def get_equivalent_numpy_forward_from_onnx(
 def get_equivalent_numpy_forward_from_onnx_tree(
     onnx_model: onnx.ModelProto,
     check_model: bool = True,
-    lsbs_to_remove_for_trees: Optional[Tuple[int, int]] = None,
+    auto_truncate=None,
```

Comment:
Not sure about the type of this, should it be a bool?

Reply:
I didn't take care of the typing in this PR (it's still in draft) and as I explained it's on hold.

`auto_truncate` is the `concrete.fhe.auto_truncate` object and not a bool, it can be None, if the feature is disabled.

In our test, we check that the feature is stable, i.e. we get the same result with and without truncate. 
(in case the concrete team makes an update that changes the behavior of truncate).